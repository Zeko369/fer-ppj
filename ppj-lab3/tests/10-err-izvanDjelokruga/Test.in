<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 x
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 1
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 n
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        BROJ 2 10
       <T_lista>
        $
      <E_lista>
       $
   <lista_naredbi>
    <naredba>
     <za_petlja>
      KR_ZA 3 za
      IDN 3 i
      KR_OD 3 od
      <E>
       <T>
        <P>
         BROJ 3 1
        <T_lista>
         $
       <E_lista>
        $
      KR_DO 3 do
      <E>
       <T>
        <P>
         IDN 3 n
        <T_lista>
         $
       <E_lista>
        $
      <lista_naredbi>
       <naredba>
        <naredba_pridruzivanja>
         IDN 4 x
         OP_PRIDRUZI 4 =
         <E>
          <T>
           <P>
            IDN 4 i
           <T_lista>
            OP_PUTA 4 *
            <T>
             <P>
              BROJ 4 3
             <T_lista>
              $
          <E_lista>
           $
       <lista_naredbi>
        $
      KR_AZ 5 az
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 6 y
       OP_PRIDRUZI 6 =
       <E>
        <T>
         <P>
          IDN 6 i
         <T_lista>
          $
        <E_lista>
         OP_PLUS 6 +
         <E>
          <T>
           <P>
            BROJ 6 1
           <T_lista>
            $
          <E_lista>
           $
     <lista_naredbi>
      $
