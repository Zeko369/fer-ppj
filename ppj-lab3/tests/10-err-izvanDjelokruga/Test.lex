IDN 1 x
OP_PRIDRUZI 1 =
BROJ 1 1
IDN 2 n
OP_PRIDRUZI 2 =
BROJ 2 10
KR_ZA 3 za
IDN 3 i
KR_OD 3 od
BROJ 3 1
KR_DO 3 do
IDN 3 n
IDN 4 x
OP_PRIDRUZI 4 =
IDN 4 i
OP_PUTA 4 *
BROJ 4 3
KR_AZ 5 az
IDN 6 y
OP_PRIDRUZI 6 =
IDN 6 i
OP_PLUS 6 +
BROJ 6 1
