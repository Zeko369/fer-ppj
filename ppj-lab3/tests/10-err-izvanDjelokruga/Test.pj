x = 1
n = 10 
za i od 1 do n
 x = i * 3    
az
y = i + 1
