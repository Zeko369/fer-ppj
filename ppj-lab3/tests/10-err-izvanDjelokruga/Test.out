3 2 n
4 3 i
err 6 i
